Makefile

```diff
@@ -111,30 +113,54 @@ docker-test-run: # Test local docker container with test fixture notebook
 # Terraform
 ####################################
 
-### Terraform-generated Developer Deploy Commands for Dev environment           ###
-dist-dev: ## Build docker container (intended for developer-based manual build)
-	docker build --platform linux/amd64 \
-	    -t $(ECR_URL_DEV):latest \
-		-t $(ECR_URL_DEV):`git describe --always` \
-		-t $(ECR_NAME_DEV):latest .
+### Terraform-generated Developer Deploy Commands for Dev environment ###
+check-arch:
```

Comment:
I like this `check-arch` command.  After some cycles with all this, might be neat to weave this into either `pre-commit`, or `make lint`, or something.